metrics/qos/evm/metrics.go

```diff
@@ -54,7 +55,7 @@ var (
 			Name:      requestsTotalMetric,
 			Help:      "Total number of requests processed by EVM QoS instance(s)",
 		},
-		[]string{"chain_id", "service_id", "request_method", "success", "error_type", "http_status_code"},
+		[]string{"chain_id", "service_id", "request_origin", "request_method", "success", "error_type", "http_status_code"},
```

Comment:
@okdas FYI on increased cardinality. 

LGTM but let's be conservative with these.

Reply:
Sure, this only has 2 distinct values.